docs/Advanced-Topics.md

```diff
@@ -62,12 +62,18 @@ you created it
-| `use_team_codes`    | `0`           | NOT YET RELEASED `1` if team codes should be display on the public pairings view, `0` if normal team names should be display                                                                                                                                                                                     |
+| `use_team_codes`    | `0`           | `1` if team codes should be display on the public pairings view, `0` if normal team names should be display                                                                                                                                                                                     |
 +---------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | `teams_public`      | `0`           | `1` if the teams list is publicly visible, `0` if not.                                                                                                                                                                                                                                                           |
 +---------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | `judges_public`     | `0`           | `1` if the judges list is publicly visible, `0` if not.                                                                                                                                                                                                                                                          |
 +---------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| `schools_public`    | `1`           | `1` if the schools are publicly visible on the public teams list.                                                                                                                                                                                                                                                |
```

Comment:
Why not get rid of this and use the `use_team_codes` setting to control this? I can't think of a circumstance where people would want to hide schools other than using team codes

Reply:
Yep, agree, check it out now.